### PR TITLE
[#139005259] Whitelist  Cabinet Office IP

### PR DESCRIPTION
**Pull request by combor**
## What

Cabinet Office would like to try the PaaS out, with a view to possibly
deploying People Finder on it.

## How to review

I would leave it for CI concourse to test but if you like you can deploy it on dev 

## Who can review

Not @combor

### CODE REVIEW

File: terraform/prod.tfvars [hunk 1]
```diff
@@ -42,6 +42,8 @@ tenant_cidrs = [
   "52.213.138.76/32",
   "52.213.151.67/32",
   "52.208.47.129/32",
+  #  Cabinet office trial
```
Comment (mtekel) [inline terraform/prod.tfvars]: why 2 spaces?

**Comment by bleach** [comment]
I confirm this was the IP address requested

File: terraform/prod.tfvars (hunk 1 above)
Comment (combor) [inline terraform/prod.tfvars]: Copy-paste left over. Also I corrected two space mistake in the commit message. Now we've got perfect two line change.

**Comment by mtekel** [comment]
LG, merging

**Pull request merged**